canvas_modules/common-canvas/src/common-properties/components/editor-form/editor-form.scss

```diff
@@ -52,15 +52,46 @@ $primary-tab-height: $spacing-08;
 	}
 }
 
-.properties-category-container {
-	position: relative;
-	&:last-of-type {
-		.properties-category-content.show {
-			border-bottom: 0;
+.properties-categories {  
+	.bx--accordion__item {
+		margin: 3 -1rem; // Remove 16px outer padding for accordion title and content
```

Comment:
I'm seeing this in the browser.

![Screenshot 2024-01-03 at 4 10 49 PM](https://github.com/elyra-ai/canvas/assets/21126000/e6af1c51-db21-4df0-b7b0-e9925e7077dd)

Reply:
Thanks for the feedback, I have removed this in my latest commit.